scripts/activate.sh

```diff
@@ -1 +1,3 @@
-python -m anaconda_anon_usage.install --enable --quiet
+pbin="${CONDA_PREFIX}/python.exe"
```

Comment:
should this be .exe here? (same below)

Reply:
I'll remove `.exe` and if that works I'm happy to have it gone

Reply:
ok - why was it here/working in the first place I'm wondering? .exes don't run on linux as far as I know or am I missing something..?

Reply:
oh yes! the reason I do that is that for noarch packages, it will run sometimes on windows and sometimes on unix. so this code is looking for the Windows executable, and if it's not there, switches over to the Unix executable

Reply:
ah thanks, I thought it was because it could be found in two different places in the prefix.